src/zenml/environment.py

```diff
@@ -27,6 +29,53 @@
 logger = get_logger(__name__)
 
 
+def get_installed_integrations() -> List[Dict[str, str]]:
+    """Returns list of installed integrations."""
```

Comment:
Either the docstring or the implementation is wrong :P
Is there any reason for having a dict here instead of just having a list of all the installed ones?

Reply:
Absolutely right! Fixed in the latest commit

Reply:
Nice! One more thing that jsut came to my mind, what do you think of having this method for the list of installed integrations on the `IntegrationRegistry` itself?